central/debug/service/diagnostics.go

```diff
@@ -141,6 +144,20 @@ func pullCentralClusterDiagnostics(ctx context.Context, filesC chan<- k8sintrosp
 	}
 }
 
+func getK8sInClusterConfig() (*rest.Config, error) {
```

Comment:
We'll need this function for the OpenShift monitoring K8s client as well to fetch the client auth CA. Maybe we should prefactor and add it somewhere in `pkg`?

Reply:
+1, `pkg/k8sutils` should be the place.